src/HealthGPS/energy_balance_model.h

```diff
@@ -11,12 +11,18 @@ namespace hgps {
 class EnergyBalanceModel final : public HierarchicalLinearModel {
   public:
     /// @brief Initialises a new instance of the EnergyBalanceModel class
-    /// @param definition The model definition instance
-    EnergyBalanceModel(EnergyBalanceModelDefinition &definition);
+    /// @param energy_equation The energy coefficients for each nutrient
+    /// @param nutrient_equations The nutrient coefficients for each food group
+    /// @param age_mean_height The mean height at all ages (male and female)
+    EnergyBalanceModel(
+        const std::unordered_map<core::Identifier, double> &energy_equation,
+        const std::unordered_map<core::Identifier, std::map<core::Identifier, double>>
+            &nutrient_equations,
+        const std::unordered_map<core::Gender, std::vector<double>> &age_mean_height);
```

Comment:
Personally, I think we should be talking about `sex` in these equations and not `gender`, which is a different thing, but I'm not going to be the one telling them what to use...

Reply:
Fair point. I suppose there won't be complaints if they don't see the change!

Reply:
Maybe leave this on the back burner then. I'll open an issue anyway.